cobra/cmd/helpers.go

```diff
@@ -196,7 +196,7 @@ func isEmpty(path string) (bool, error) {
 		if err != nil {
 			return false, err
 		}
-		list, err := f.Readdir(-1)
```

Comment:
Good question. I guess the compiler allows it in that particular situation? Note: I think I actually found this through gometalinter and not golint.